client/api/editor/item_events.ts

```diff
@@ -137,6 +137,10 @@ export function getEventsInstance(type: EDITOR_TYPE): IEditorAPI['item']['events
             return;
         }
 
+        if (updates && original?.version == original?.posted_version) {
+            original['version'] = original?.version ? original['version'] + 1 : 1;
```

Comment:
isn't there any version field that would be handling it already? like `_current_version` ?

Reply:
I see, maybe we can use the `versioncreated` then and have `versionposted` so we wouldn't have to maintain the version manually, there might be more places when it should be updated and it might get out of sync, like when it would get updated via ingest

Reply:
Okay let me try , Thanks !